### ISSUE TITLE
[question] Receiving mavlink msgs via a ros node?

### ISSUE THREAD

**Issue by dkati** [mavlink/mavros#1637]
Good evening. 
Actually, i have 2 questions

**(1)**

Is this possible? i want to read mavlink messages in a cpp ros node but i cant understand a thing...

(i am already flying with offboard mode, cpp waypoints handling etc...)

To be more specific, i want to read the messages that be spawned by 
```
rosrun mavros mavros_node
```
but also from
```
make px4_sitl gazebo
```

I think the msgs from mavros_node can be grabbed from /rosout subscription at 'msg' section (?)

-----------------------------------------------------------------------------------------------------------------------------------------

My main goal is to implement a mechanism of "did the mission sending succeded?".
**(2)**

Do i need https://mavlink.io/en/messages/common.html#MISSION_ACK ? if so, i see no example of implementing it

**Comment by vooon** [comment]
No, that's not possible to get mavlink messages sent by mavros to fcu via ros messages because it talks directly to the device.
(well that is possible if you put some proxy, like in mavros 2).

But isn't that enough to check success flag of WaypointPush.srv?

**Comment by dkati** [comment]
any example or something for that waypoint plugin?
